src/components/genre-select.tsx

```diff
@@ -0,0 +1,37 @@
+import React, {useState} from "react";
+
+interface GenreSelectProps {
+    genres: string[];
+    selectedGenre: string;
+    onSelect: (genre: string) => void;
+}
+
+export const GenreSelect: React.FC<GenreSelectProps> = ({genres = [], selectedGenre, onSelect}) => {
+    const [selected, setSelected] = useState(selectedGenre);
+    const handleSelect = (genre: string) => {
+        setSelected(genre);
+        onSelect(genre);
+    }
+
+    return (
+        <nav className="flex gap-8">
+            {genres.map((genre) => (
+                <button
+                    key={genre}
+                    onClick={() => handleSelect(genre)}
+                    className={`
+            text-white text-xl uppercase py-4 px-2 relative
+            hover:after:bg-red-600 hover:after:absolute hover:after:h-1 hover:after:w-full hover:after:bottom-0 hover:after:left-0
+            ${
+                        selected === genre
```

Comment:
It looks like there is something wrong with indentation, it is not consistent. 
I suggest that you use prettier, add a configuration file to your project and reformat the code in accordance with the config.